### PR TITLE
Biome and feature work, a little bit of villager stuff

### PR DESCRIPTION
**Pull request by Prospector**

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 1]
```diff
@@ -1,39 +1,59 @@
...
 	METHOD cv getDeathSound ()Lxm;
 	METHOD d getVillagerWorld ()Lbbp;
+	METHOD dB isTrading ()Z
 	METHOD dD getInventory ()Lafx;
+	METHOD dJ ()V
 	METHOD dM ()Laqs;
 	METHOD e getVillagerPos ()Let;
 	METHOD e getHurtSound (Lage;)Lxm;
 	METHOD n ()V
+	METHOD v setWilling (Z)V
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Could this be a tad more descriptive?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 1 above)
Comment (Prospector) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: I mean the field and nbt tag are called willing

Comment (asiekierka) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Sure

File: mappings/net/minecraft/world/gen/feature/Feature.mapping [hunk 2]
```diff
@@ -29,3 +29,5 @@ CLASS bvg net/minecraft/world/gen/feature/Feature
 		ARG 3 random
 	METHOD a deserialize (Lcom/mojang/datafixers/Dynamic;)Lbvh;
 	METHOD a register (Ljava/lang/String;Lbvg;)Lbvg;
+	METHOD e getMonsterSpawnList ()Ljava/util/List;
```
Comment (Gegy) [inline mappings/net/minecraft/world/gen/feature/Feature.mapping]: `getMonsterSpawns`?

**Pull request closed without merging**